docs/components/select.md

```diff
@@ -1,1013 +1,741 @@
-      <div class="select-item-label">{{ item.label }}</div>
-      <div class="select-item-desc">{{ item.label }}</div>
+    <template #item-template="{ item }">
+      <KongIcon />
+      {{ item?.label }}
     </template>
   </KSelect>
 </ClientOnly>
 
 ```html
 <KSelect
-  autosuggest
-  :items="items"
-  :loading="loading"
-  width="300px"
-  appearance="select"
-  @query-change="onQueryChange"
+  :items="selectItems"
+  reuse-item-template
 >
-  <template v-slot:item-template="{ item }">
-    <div class="select-item-label">{{ item.label }}</div>
-    <div class="select-item-desc">{{ item.label }}</div>
+  <template #item-template="{ item }">
+    <KongIcon />
+    {{ item?.label }}
   </template>
 </KSelect>
-
-<script>
-function debounce(func, timeout) {
-  let timer;
-  return function (...args) {
-    clearTimeout(timer);
-    timer = setTimeout(() => {
-      func.apply(this, args);
-    }, timeout);
-  };
-}
-const allItems = new Array(10).fill().map((_, i) => ({
-  label: `Item ${i}`,
-  description: `This is the description for item ${i}.`,
-  value: `autosuggest-item-${i}`,
-  ...(i > 5 && { group: `${i % 2 === 0 ? 'Even items greater than 5' : 'Odd items greater than 5'}` })
-}));
-export default {
-  data() {
-    return {
-      defaultItems: [],
-      items: [],
-      loading: true,
-    }
-  },
-  methods: {
-    onQueryChange (val) {
-      if (val === '' && !this.defaultItems.length) {
-        // If query is empty and default items are not fetched, fetch them
-        this.loading = true;
-        setTimeout(() => {
-          this.defaultItems = allItems;
-          this.items = this.defaultItems.map(item => Object.assign({}, item));
-          this.loading = false;
-        }, 200);
-        return;
-      }
-      if (val === '') {
-        // If query is empty and default items are fetched, use the default items
-        this.items = this.defaultItems.map(item => Object.assign({}, item));
-        return;
-      }
-      this.debouncedHandler(val);
-    },
-    debouncedHandler: debounce(function (val) {
-      this.loading = true;
-      // Fetch items that contain the keyword
-      setTimeout(() => {
-        this.items =
-          allItems
-            .filter(item => item.label.toLowerCase().includes(val.toLowerCase()) || item.description.toLowerCase().includes(val.toLowerCase()))
-            .map(item => Object.assign({}, item));
-        this.loading = false;
-      }, 200);
-    }, 400)
-  }
-}
-</script>
 ```
 
-### loading
+### positionFixed
 
-When `autosuggest` is enabled, you can use the `loading` prop to show a loading indicator while fetching data from API.
-By default, the loading indicator is a spinner icon, and you can implement your own indicator using the `loading` slot.
-See [autosuggest](#autosuggest) for an example.
+Use fixed positioning of the popover to avoid content being clipped by parental boundaries. See [KPopover](/components/popover#positionfixed) docs for more information.
```

Comment:
```suggestion
Use fixed positioning for the select dropdown to avoid content being clipped by the boundaries of its parent container. See [KPopover](/components/popover#positionfixed) docs for more information.
```